src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrProxy.cs

```diff
@@ -48,30 +43,25 @@ public ValidationFailure Test(NotifiarrSettings settings)
                 SendNotification(variables, settings);
                 return null;
             }
-            catch (HttpException ex)
+            catch (NotifiarrException ex)
             {
-                if (ex.Response.StatusCode == HttpStatusCode.Unauthorized)
-                {
-                    _logger.Error(ex, "API key is invalid: " + ex.Message);
-                    return new ValidationFailure("APIKey", "API key is invalid");
-                }
-
-                _logger.Error(ex, "Unable to send test message: " + ex.Message);
-                return new ValidationFailure("APIKey", "Unable to send test notification");
+                return new ValidationFailure("APIKey", ex.Message);
             }
             catch (Exception ex)
             {
-                _logger.Error(ex, "Unable to send test notification: " + ex.Message);
-                return new ValidationFailure("", "Unable to send test notification");
+                _logger.Error(ex, ex.Message);
+                return new ValidationFailure("", "Unable to send test notification. Check the log for more details.");
             }
         }
 
         private void ProcessNotification(StringDictionary message, NotifiarrSettings settings)
         {
             try
             {
-                var requestBuilder = new HttpRequestBuilder(URL).Post();
-                requestBuilder.AddFormParameter("api", settings.APIKey).Build();
+                var instanceName = _configFileProvider.InstanceName;
+                var requestBuilder = new HttpRequestBuilder(URL + "/api/v1/notification/prowlarr/").Post();
```

Comment:
Can we get rid of this trailing `/` or is it needed? Also does this allow us to kill the cleanse regex we added specifically for these?

Reply:
@austinwbest @davidnewhall defer to y'all on if the / is needed

Yes can purge the regex then

Reply:
You can prolly simplify this now since there is no dynamic domains or data and just set this to `https://notifiarr.com/api/v1/notification/prowlarr` and remove the URL variable. Same could be said for the instance name, just set it to `_configFileProvider.InstanceName` explicitly and dump the variable

Reply:
no regex kill needed. it was aspecific cleanse added that would also cover generic webhooks